In-house patch
This is the original patch turned to unified-diff format only.
It was created during the sfwnv -> Userland conversion
...but even the author of the patch (Vlad) do not remember its meaning.
It is safe to leave it here.

--- a/lib/Foomatic/DB.pm	2016-07-20 00:40:31.000000000 -0700
+++ b/lib/Foomatic/DB.pm	2016-07-20 13:59:40.150491933 -0700
@@ -111,7 +111,7 @@
     }
 
     # Build a new overview
-    my $otype = ($cupsppds ? '-C' : '-O');
+    my $otype = ($cupsppds ? '-C' : '-C');
     $otype .= ' -n' if ($cupsppds == 1);
     # Generate overview Perl data structure from database
     my $VAR1;
